pkg/router/controller/route_secret_manager.go

```diff
@@ -28,18 +38,33 @@ type RouteSecretManager struct {
 
 	secretManager secretmanager.SecretManager
 	secretsGetter corev1client.SecretsGetter
+	routelister   routelisters.RouteLister
 	sarClient     authorizationclient.SubjectAccessReviewInterface
+	// deletedSecrets tracks routes for which the associated secret was deleted after intial creation of the secret monitor.
```

Comment:
```suggestion
	// deletedSecrets tracks routes for which the associated secret was deleted after initial creation of the secret monitor.
```

Reply:
Thanks for the catch. Updated in https://github.com/openshift/router/commit/62daef459fde17c27007cd8d0ac6bc24209b2120